### PR TITLE
DISPLAY-1030: Updated development documentation

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/DISPLAY-1030

### CODE REVIEW

File: development/development.md [hunk 1]
```diff
@@ -14,3 +14,140 @@ The templates should be installed with the `-dev.json` file. This is done with t
...
+
+```shell
+cd display-admin-client
+
+# Create config
+cat public/example_config.json | docker run --rm --interactive efrecon/jq:1.7 --arg API "http://$(docker compose --project-name displayapiservice port nginx 8080)" '.api = $API' > public/config.json
+docker compose up --detach
+
+docker compose run --rm node yarn install
+docker compose run --rm node bash -c 'cd src/redux/api && npm install && npm start'
```
Comment (tuj) [inline development/development.md:110]: This is only needed fo rebuilding the redux API endpoints when the api.json OpenAPI spec has been changed.

File: development/development.md [hunk 2]
```diff
@@ -14,3 +14,140 @@ The templates should be installed with the `-dev.json` file. This is done with t
...
+docker compose run --rm node yarn install
+docker compose run --rm node yarn build
+
+# Set config
+cat public/example_config.json | docker run --rm --interactive efrecon/jq:1.7 --arg API "http://$(docker compose --project-name displayapiservice port nginx 8080)" '.apiEndpoint = $API|.authenticationEndpoint = $API+"/v1/authentication/screen"|.authenticationRefreshTokenEndpoint = $API+"/v1/authentication/token/refresh"|.dataStrategy.config.endpoint = $API' > public/config.json
+cat public/example_release.json | docker run --rm --interactive efrecon/jq:1.7 '.' > public/release.json
+
+open "http://$(docker compose port nginx 8080)"
+
+open "http://$(docker compose --project-name display-admin-client port nginx 8080)/admin/screen/list"
```
Comment (tuj) [inline development/development.md:134]: Why open this? Maybe add a comment

**Review by tuj — CHANGES_REQUESTED** [review]

File: development/development.md (hunk 1 above)
Comment (rimi-itk) [reply development/development.md:110]: I'll remove. I don't understand it anyway.

File: development/development.md (hunk 2 above)
Comment (rimi-itk) [reply development/development.md]: I've elaborated a little on this.

**Review by tuj — APPROVED** [review]

**Pull request merged**